pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -122,7 +129,7 @@ func newClaimInfoFromResourceClaim(resourceClaim *resourcev1alpha2.ResourceClaim
 		resourceHandles = make([]resourcev1alpha2.ResourceHandle, 1)
 	}
 
-	return newClaimInfo(
+	return *newClaimInfo(
```

Comment:
For consistency, should we update `newClaimInfo()` to return a non-pointer type as well?

Reply:
done